lib/vanagon/platform/windows.rb

```diff
@@ -57,6 +57,16 @@ def generate_msi_packaging_artifacts(workdir, name, binding)
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "wix",  "#{name}.wxs"), false, { :binding => binding })
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service.component.wxs.erb"), File.join(workdir, "wix", "service.#{name}.wxs"), false, { :binding => binding })
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.filter.xslt.erb"), File.join(workdir, "wix", "#{name}.filter.xslt"), false, { :binding => binding })
+
+        # Project Specific Wix Files (ERB Templates)
+        ["customactions","properties","sequences"].each do |wix_filename|
```

Comment:
Correct for the custom actions - I haven't tested the other scenarios.

Was wondering though about allow us to "Feed" in or modify the list of files to be processed.
At the moment, we are making the assumption (at the candle side) that all the files of type *.wxs will be processed.
So we could:

1. Simply glob for all the *.wxs.erb files and translate them (using your idea last week of keeping their names), or
2. Add directive to allow us to specify the files to be included on the project specific side, similar to the patch directive we already use.